src/components/Remark.tsx

```diff
@@ -39,6 +44,7 @@ const Remark = ({ className, targetChain }: Props) => {
   const [remarkInput, setRemarkInput] = useState('0x');
   const lane_id = useLaneId();
   const [executionStatus, setExecutionStatus] = useState('');
+  const { account: currentAccount } = useAccountContext();
```

Comment:
```suggestion
  const { account } = useAccountContext();
```

Reply:
This was addressed already in a new PR... but yes ... that does not makes sense.